src/main/java/uk/gov/hmcts/reform/sscs/callback/handlers/DwpUploadResponseHandler.java

```diff
@@ -50,30 +50,30 @@ public void handle(CallbackType callbackType, Callback<SscsCaseData> callback) {
         }
 
         BenefitType benefitType = callback.getCaseDetails().getCaseData().getAppeal().getBenefitType();
-        Benefit benefit = Benefit.getBenefitByCodeOrThrowException(benefitType.getCode());
+        SscsType benefitSscsType = callback.getCaseDetails().getCaseData().getBenefitType().get().getSscsType();
```

Comment:
Shouldn't really be using a `.get()` on an optional as if it doesn't exist the code will blow up. 
You can either use the `.isPresent()` first to make sure it is there? 
Or I used this same getBenefitType() method in a PR yesterday and used a filter, maybe you can do something similar: https://github.com/hmcts/sscs-tribunals-case-api/pull/2544/files#diff-2d54adff198e2b14178075e21431bc2410f21b47f06711fe5661e07786112f01R79